### ISSUE TITLE
Final issues CS211

### ISSUE THREAD

**Issue by houshuang** [chili-epfl/FROG#1071]
- [x] Next activity wipes projector view, dashboard overlapping (Stian)
- [x] Crash in ac-train stats, add Flow definitions (Louis)
- [x] ac-train dashbord stats needs axis labels, and don't repeat numbers (Prastut)
- [x] train interface intro screen, font too small? 
- [x] bug in command help - modal too small (Prastut)
- [x] look at command interface/graphical data and think about instructions, and if we are correctly matching correctness (Jenny)
- [x] ac-ranking prompt: make them harder to ignore, salient... difference expert/novice, image? take out images of interfaces for group ac1 and ac2 (Jenny)
- [x] take out images of interfaces for group ac1 and ac2 (Jenny)
- [x] label: graphical -> map. (Prastut: train)
- [x] label: graphical -> map. (Jenny: ranking)
- [ ] one student was not matched with anyone, robot said "undefined" (Wafa) - look at data (Stian/Louis)
- [x] ranking feedback if group has not agreed - move/make more salient (Jenny)
- [x] round time on tooltips ac-train (Prastut)
- [x] prediction went negative, add Flow types to mergeLog and Viewer (Jiaxi)
- [ ] Firefox: ac-ranking, big blank square
- [x] add blank dashboard (Stian)
- [x] remove student stats from teacher view (Louis)
- [x] remove buttons from orchestration view (Louis)

From old issue:
- [x] Add multi-server support to new dashboard approach (Stian)
- Want a graph showing the correlation between performance and preference (CANCELED)

**Comment by lfaucon** [comment]
Added items from old todo list

**Comment by houshuang** [comment]
@prastut I'm guessing you've fixed the axis stuff with your PR?

**Comment by prastut** [comment]
@houshuang I have. Forgot to check it here.

**Comment by houshuang** [comment]
I could not reproduce the Firefox issue, no idea what to do without more info.

**Comment by houshuang** [comment]
@lfaucon I know you found the issue with Wafa, have you done anything with the operator to prevent it from matching people who didn't even begin the ac-ranking (or at least match them with other people who didn't begin)? If they leave after they've begun ranking, of course, there's not much we can do for now.

**Comment by lfaucon** [comment]
@houshuang 
Students with missing data will be at distance "0" of all other students, that means that the algorithm will try not to match them with students that could have better matches when it can. "0" is a good default value when we run a maximum distance algorithm, but we will want "infinity" if we plan on matching similar students (maybe that could be a configuration of op-xy-distance).

I think we should merge #1099 which fix the broken messages when data is missing

**Comment by Jiaxigu** [comment]
I checked the log from Thursday and no negative value could be reproduced. I believe it was a rendering problem from Victory.

For the drop in the progress curve, it happens when student clicked 'return' after submitting answers. (Progress == 0.5 when all questions completed, progress == 1 when submitted)

![wechatimg120](https://user-images.githubusercontent.com/14288948/39427286-89a1138a-4c83-11e8-8f3a-6b6ddadc8016.jpeg)

TL;DR: I believe the prediction curves are ready for tomorrow =)
